universal-application-tool-0.0.1/app/controllers/admin/AdminApplicationController.java

```diff
@@ -43,20 +45,23 @@ public AdminApplicationController(
       ProgramApplicationListView applicationListView,
       ProgramApplicationView applicationView,
       ApplicationRepository applicationRepository,
+      ProfileUtils profileUtils,
       Clock clock) {
     this.programService = checkNotNull(programService);
     this.applicantService = checkNotNull(applicantService);
     this.applicationListView = checkNotNull(applicationListView);
+    this.profileUtils = checkNotNull(profileUtils);
     this.applicationView = checkNotNull(applicationView);
     this.applicationRepository = checkNotNull(applicationRepository);
     this.clock = clock;
     this.exporterService = checkNotNull(exporterService);
   }
 
-  @Secure(authorizers = Authorizers.Labels.UAT_ADMIN)
-  public Result downloadAll(long programId) {
+  @Secure(authorizers = Authorizers.Labels.ANY_ADMIN)
```

Comment:
question (blocking): (here and below) - should we be catching the `SecurityException` that is thrown and returning `unauthorized()`?

Reply:
A SecurityException is automatically translated into Unauthorized but you're right that we could get a little more control that way and be a bit more explicit.  :)  Done.